app/views/certificate_templates/_form.html.haml

```diff
@@ -1,11 +1,12 @@
 = simple_nested_form_for @certificate_template do |f|
   = render 'shared/error_messages', object: f.object
-
+  
   .form-inputs
     = f.input :title
     = f.input :file, wrapper: :vertical_file_input, as: :file
     = f.association :institution, include_blank: false
     = f.input :program_type, collection: CertificateTemplate.program_types.keys, include_blank: false
+    
```

Comment:
Please do not commit blank spaces in case those are only changes made to the file

Reply:
I'm sorry.

Reply:
I do not see it corrected

![Screenshot 2024-12-05 at 13 43 50](https://github.com/user-attachments/assets/f0908d13-217d-457a-a581-82b448b5118c)

Reply:
Returned the file to the state it is in in the master branch